src/main/java/command/UpdateCommand.java

```diff
@@ -0,0 +1,89 @@
+package command;
+
+import exception.DukeException;
+import exception.DukeUnknownException;
+import storage.Storage;
+import task.*;
+import ui.Ui;
+
+public class UpdateCommand extends Command {
```

Comment:
first glance I thought this is a command for Add new commands, so that there will be a data structure that holds all the commands, and others programmer can create new sub class command and add to command map elsewhere. 🤔